Analyticsv3Sample/README.md

```diff
@@ -0,0 +1,50 @@
+
+## DAIMTO Google Analytics API Sample for .NET  ##
+
+View and manage your Google Analytics data
+
+## Description ##
+The DAIMTO Google Analytics API Sample for .NET instructs you in the usage of the Google APIs client library for .NET.
+
+The sample contains OAuth2.0 authentication, service account, and and public API.   As well as methods for each of the Resources avaliable in the Analytics API.  
+
+Tutorials for some of these sample projects can be found on my website [Daimto.com](http://www.daimto.com)
+
+Requirements 
+=================================
+* [.NET Framework 4.0](http://www.microsoft.com/en-us/download/details.aspx?id=17851)
+* [.NET Framework 4.5](http://www.microsoft.com/en-us/download/details.aspx?id=30653)
+* [Nuget package Google Apis Analytics v3](https://www.nuget.org/packages/Google.Apis.analytics.v3/)
+  *   PM> Install-Package Google.Apis.Analytics.v3
+
+SETUP
+=================================
+
+Make sure you create your own Oauth credentials at https://console.developers.google.com/.
+
+Walk though on how to create the diffrent credentails can be found:
+* [Google Developer console service account](http://www.daimto.com/google-developer-console-service-account/)
```

Comment:
Just something you want to notice.
The Developers Console API was changed lately, make sure that the tutorial are still up to date.